### ISSUE TITLE
No Reload Decorators not work

### ISSUE THREAD

**Issue by jindili** [reloadware/reloadium#92]
# environments
### Reloadium version:
Plugin version:0.9.5
Package version:0.9.10

### Python version: 
Python 3.10 on Windows 10 64bit

# Simple Code used to test reloadium
reference from
Documentation - Decorators | Reloadium
https://reloadium.io/documentation/decorators
```
def cal():
    # reloadium: no_reload
    print("only once in function")
    a = 1
    b = 4
    result = a + b
    return  result

if __name__ == '__main__':
    # reloadium: no_reload
    print("only once in main")

    print("Reload 1a")
    print("Reload 2")
    cal()
```
![image](https://user-images.githubusercontent.com/73845/215326823-5a6a71b0-6ad8-4102-94fe-35d327ab8277.png)

# Bug 
Decorator in main and in function both not work.
Code under no_reload decorator all reloaded on code change.

**Comment by dkrystki** [comment]
Thanks for submitting this.
It's already fixed but will have to wait for the next release.

Until then the workaround is to use `# reloadium: no_reload` inline like this:

```
def cal():
    print("only once in function")   # reloadium: no_reload
    a = 1
    b = 4
    result = a + b
    return  result

if __name__ == '__main__':
    print("only once in main")   # reloadium: no_reload

    print("Reload 1a")
    print("Reload 2")
    cal()
```

**Comment by jindili** [comment]
The workaround works in function but not in main

## Code
```
def cal():
    print("reload only once in function")   # reloadium: no_reload
    a = 1
    b = 4
    print("Reload 1d in function")
    result = a + b
    return  result

if __name__ == '__main__':
    print("reload only once in main")   # reloadium: no_reload

    print("Reload 1d in main")
    print("Reload 2 in main")
    cal()
```

## Breakpoint setup
![image](https://user-images.githubusercontent.com/73845/215371518-81c85e26-0dbb-491d-9bcd-5aefb83a7c85.png)


## Debugging Rersult
![image](https://user-images.githubusercontent.com/73845/215371481-9bd8317c-0b34-4ff1-913a-f717767865f4.png)

**Comment by dkrystki** [comment]
Fixed in reloadium 0.9.11 and PyCharm plugin 0.9.6
